integration/integration_test.go

```diff
@@ -29,7 +29,8 @@ var (
 	// image such as docker.io/kinvolk/gadget:latest
 	image = flag.String("image", "", "gadget container image")
 
-	doNotDeploy = flag.Bool("no-deploy", false, "don't deploy Inspektor Gadget")
+	doNotDeployIG  = flag.Bool("no-deploy-ig", false, "don't deploy Inspektor Gadget")
+	doNotDeploySPO = flag.Bool("no-deploy-spo", false, "don't deploy the Security Profiles Operator (SPO)")
```

Comment:
Instead of `*DeploySPO`, can't we call this `*InstallSPO`?
So, we are not bothered regarding IG.

Reply:
Then why not "InstallIG"? I think both of them should have the prefix.